Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -531,12 +531,41 @@ def upload_indicators_list_request(self, reference_name, indicators_list):
         )
 
     def get_custom_fields(
-            self, limit: Optional[int] = None) -> List[dict]:
+            self, limit: Optional[int] = None, field_name: Optional[List[str]] = None,
+            likes: Optional[List[str]] = None, _filter: Optional[str] = None, fields: Optional[List[str]] = None)\
```

Comment:
lets stick to PEP8 conventions and use a trailing underscore `filter_`
https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles